### PR TITLE
added nightfox theme

### PR DESCRIPTION
**Pull request by papatenko**
Also, something seemed broken with the table so that's fixed as well.

### CODE REVIEW

File: schemes.yaml [hunk 1]
```diff
@@ -1007,6 +1007,32 @@ schemes:
...
+      red:     '0xd16983'
+      green:   '0x8ebaa4'
+      yellow:  '0xe0c989'
+      blue:    '0x86abdc'
+      magenta: '0xbaa1e2'
+      cyan:    '0x7ad5d6'
+      white:   '0xe4e4e5'
+    indexed_colors:
+      - { index: 16, color: '0xf4a261' }
+      - { index: 17, color: '0xd67ad2' }
```
Comment (chrisduerr) [inline schemes.yaml:1034]: This seems rather unusual. What motivation is behind overwriting these indexed colors?

**Review by chrisduerr — CHANGES_REQUESTED** [review]

File: schemes.yaml (hunk 1 above)
Comment (papatenko) [reply schemes.yaml:1034]: Oh, that isn't my doing. I copied from the alacrity color scheme from this [file](https://github.com/EdenEast/nightfox.nvim/blob/main/extra/nightfox/nightfox_alacritty.yml) provided by the nightfox repo. **I'm not sure what the motivation is behind it is**, as alacritty looks the same with or without indexed colors. I noticed that other themes contained lines to alter those index colors as well so I did not think much of it.

Comment (chrisduerr) [reply schemes.yaml:1034]: Unless you have a good reason for it, I'd rather remove it.

Comment (papatenko) [reply schemes.yaml:1034]: Understood, the three lines have been removed.

**Review by chrisduerr — APPROVED** [review]

**Pull request merged**